### DISCUSSION TITLE
Suggestion for 'Edit' view for Chord Chart Studio

### DISCUSSION THREAD

**Discussion by sgipianobuddha** [no-chris/chord-chart-studio#631] [Ideas]
I'm really enjoying your app, @no-chris.
I'm going through my cover-songbook that is currently pencil/paper and upgrading it.
I'm getting faster at importing lyrics and applying the ChordMark code.
One thing that's slowing me down, however, is that the layout I use for the finished (pdf) product isn't like your examples. Whereas your examples contain short lyric lines (like a poem) that extend max half-page, I like longer lyric lines that go all the way across the page (like prose).
I'm finding it slow-going because I need to toggle between 'Edit' view and 'Print/PDF Preview' to make sure the code I'm inputting is coming out the way I'd like.
I wish you had an option to hide the "Result Preview" column in the 'Edit' view. This way, my code would extend across the page farther, instead of wrapping to the half-page column.
The two columns in 'Edit' view also slow me down, because whenever I toggle back from Print/PDF Preview' view to 'Edit' view, the coding column resets to the top, which means I must scroll down and find my place each time; this takes the most time!
Thank you for considering my suggestion.
<img width="757" alt="My 'Edit' view example" src="https://github.com/no-chris/chord-chart-studio/assets/148682967/195ccb39-dbf9-4196-8041-6e2cc029a2ff">
<img width="791" alt="My 'Print:PDF Preview' render" src="https://github.com/no-chris/chord-chart-studio/assets/148682967/678c936f-6553-446a-b48b-20dfa7e598c2">

**Comment by no-chris** [comment]
Thanks for the suggestion. This sounds like a reasonable feature request and rather easy to implement. I cannot commit to a timeframe for implementing it though, as I have very little available free time on my hand currently.

But I can think of a workaround, if you are able to add a custom CSS rule to Chord Chart Studio then it should be easy enough to hide. After a quick search it seems that there is a Chrome extension that is doing just that: https://chrome.google.com/webstore/detail/stylish-custom-themes-for/fjnbnpbmkenffdnngjfgmeleoegfcffe?hl=en
I haven't tried it myself though.

If it works, the following rules should do the trick:
```css
.songEditor-source {
    flex: 1 1 100%;
}
.songEditor-preview {
    display: none;
}
```

Let me know if you need help adding them

**Comment by sgipianobuddha** [comment]
Thanks Christophe!
Unfortunately coding/programing isn’t my strong suit.
I’ll just wait for you to find time to make the update.
Cheers,
Sam

**Reply by no-chris** [reply]
Really, it's no big deal and you don't need developer knowledge.

Just install the extension and copy/paste the styles in the editor with the current tab opened on Chord Chart Studio

<img width="1184" alt="Screenshot 2023-10-30 at 20 57 59" src="https://github.com/no-chris/chord-chart-studio/assets/11467480/0c7c1f38-b7b8-496b-8e1f-0764ceec2dac">

**Comment by sgipianobuddha** [comment]
Ok thanks! 
I’ve tried it and got it to work in Chrome. 
Trouble is, I have been using the app mostly in my Safari browser, so I’ve got a bunch of charts already input in Safari. 
How can I…
1) Do the same (or similar) extension (Stylish) in Safari 17, or
2) Transfer the charts I’ve input via Safari 17 into my Chrome browser?
Thanks again,
SAM
